idaplugin/rematch/actions/match.py

```diff
@@ -34,9 +34,17 @@ def perform(self):
       i, offset = self.function_gen.next()
 
       func = instances.FunctionInstance(self.file_id, offset)
-      network.query("POST", "collab/instances/", params=func.serialize(),
-                    json=True)
-
+      try:
+        network.query("POST", "collab/instances/", params=func.serialize(),
+                      json=True)
+      except exceptions.QueryException as e:
```

Comment:
It's already covered in a higher try, should we raise specifically for the network part ?